uv_ssl_t.gyp

```diff
@@ -1,8 +1,8 @@
 {
   "variables": {
     "gypkg_deps": [
-      "git://github.com/libuv/libuv.git@^1.9.0 => uv.gyp:libuv",
-      "git://github.com/indutny/uv_link_t@^1.0.0 [gpg] => uv_link_t.gyp:uv_link_t",
+      "git://github.com/libuv/libuv.git@^1.12.0 => uv.gyp:libuv",
+      "git://github.com/indutny/uv_link_t@^1.0.4 [gpg] => uv_link_t.gyp:uv_link_t",
```

Comment:
Done. So, how does the `^1.9.0` sematics work and why the exat value of `1.9.0` is here? Thx